p2p/src/net/default_backend/peer.rs

```diff
@@ -139,14 +139,15 @@ where
         // with the same invalid clock (while honest nodes can't connect).
         // After that, the node is open to all kinds of attacks.
 
-        // We cannot know the time of the remote peer at the moment of recording the timestamp,
-        // hence, we do the following: We assume that the remote peer's time is somewhere in the middle,
-        // between the time when the message was sent and the time when the message was received.
-        // This is a reasonable assumption that requires:
-        // 1. The latency in sending and receiving the message is the same
-        // 2. The time required to do computations on the message to create the response is negligible
-        // The formula for the local time at the moment the remote time was recorded is:
-        //     remote_time = local_time_start + (local_time_end - local_time_start) / 2
+        // We do not know at what point point exactly the peer recorded its timestamp. However, we
```

Comment:
I was looking for a typo. Wasn't disappointed 😄

Reply:
> I was looking for a typo. Wasn't disappointed 😄

Heh thanks. Always happy to include a handful of them :laughing: